package.json

```diff
@@ -48,6 +48,7 @@
     "eslint": "3.18.0",
     "eslint-plugin-google-camelcase": "0.0.2",
     "express": "4.14.0",
+    "fetch-mock": "^5.12.1",
```

Comment:
Because you upgraded the `yarn.lock` to use unsupported versions. Revert changes to `yarn.lock`, then run `yarn install` and it'll work again.